### ISSUE TITLE
Individual Feedback Assessment Technique

### ISSUE THREAD

**Issue by StevenClontz** [RunestoneInteractive/rs#10]
**Is your feature request related to a problem? Please describe.**
I'd like Runestone to implement http://www.if-at.com/home/about/default.aspx, a popular assessment mechanism in Team-Based Learning.

**Describe the solution you'd like**
My current applet https://scratchee.clontz.org/alpha.1/#/design mocks up the basic functionality (aside from assocating responses to teams/students and persisting results). This is a Svelte app: https://github.com/StevenClontz/scratchee

**Describe alternatives you've considered**
It's possible this might be implemented as a configuration for peer instruction(?).

**Additional context**
Brad and I discussed in Zoom today. Feel free to assign this issue to me to work on this spring (though I'd appreciate guidance and some help scaffolding a dev environment).

**Comment by bnmnetp** [comment]
Closing for lack of activity. @StevenClontz offered to build this in spring 2023 and it didn't happen — no criticism intended, that's just how volunteer time goes — and nobody else has picked it up in the three years since.

The idea is still a good one, and the scratchee prototype is a real head start if you or anyone else wants to return to it. Happy to reopen if so.

**Comment by StevenClontz** [comment]
Agreed and no worries. Thanks for all you do for the community Brad
